source/_integrations/motionblinds_ble.markdown

```diff
@@ -11,6 +11,7 @@ ha_codeowners:
   - '@jerrybboy'
 ha_config_flow: true
 ha_platforms:
+  - button
```

Comment:
Could add a description of the button functionality that was added? 
Something like this, for inspiration:
https://github.com/home-assistant/home-assistant.io/blob/7b9db946866a38e591b439343c0d23017f5c14df/source/_integrations/plex.markdown?plain=1#L105-L141

Reply:
> Could add a description of the button functionality that was added? Something like this, for inspiration:
> 
> https://github.com/home-assistant/home-assistant.io/blob/7b9db946866a38e591b439343c0d23017f5c14df/source/_integrations/plex.markdown?plain=1#L105-L141

Sure fixed it the same way I did for select and sensor.